src/hooks/useCryptocurrencies.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import { useState, useEffect } from 'react'
+import type { CryptoCurrency } from '@ledgerhq/live-common/lib/types'
+
+import { listCryptoCurrencies } from 'config/cryptocurrencies'
+import useEnv from 'hooks/useEnv'
+
+const useCryptocurrencies = (onlyTerminated: boolean = false): ?(CryptoCurrency[]) => {
+  const devMode = useEnv('MANAGER_DEV_MODE')
+  const [cryptos, setCryptos] = useState(() => listCryptoCurrencies(devMode, onlyTerminated))
```

Comment:
this state is not required. you can just `const cryptos = listCryptoCurrencies(devMode, onlyTerminated)`

Reply:
yeah but if we switch devMode off don't we need to recalculate the cryptocurrencies ?

Reply:
:magic: of hooks